### PR TITLE
Fix region migration IT & Add daily IT

### PR DESCRIPTION
**Pull request by liyuheng55555**

### CODE REVIEW

File: integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java [hunk 1]
```diff
@@ -205,10 +205,11 @@ public void generalTestWithAllOptions(
     EnvFactory.getEnv().registerDataNodeKillPoints(new ArrayList<>(dataNodeKeywords));
     EnvFactory.getEnv().initClusterEnvironment(configNodeNum, dataNodeNum);
 
-    try (final Connection connection = EnvFactory.getEnv().getConnection();
-        final Statement statement = connection.createStatement();
-        SyncConfigNodeIServiceClient client =
-            (SyncConfigNodeIServiceClient) EnvFactory.getEnv().getLeaderConfigNodeConnection()) {
+    try {
+      final Connection connection = EnvFactory.getEnv().getConnection();
```
Comment (Caideyipi) [inline integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java:209]: Better close them after use....

Comment (liyuheng55555) [reply integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java:209]: good point, thank you cdyp

**Review by Caideyipi — APPROVED** [review]
LGTM

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**